lean4-infoview/src/infoview/rpcInterface.ts

```diff
@@ -74,6 +74,7 @@ export interface InteractiveGoal {
     hyps: InteractiveHypothesis[]
     type: CodeWithInfos
     userName?: string
+    goalPrefix: string
```

Comment:
```suggestion
    goalPrefix?: string
```
The property is not present in older Lean versions (or any Lean version yet for that matter).

Reply:
To elaborate, the issue is that while we can expect users to always have an up-to-date `vscode-lean4` thanks to the auto-update mechanism, they might be using arbitrary server versions, so we commit to supporting any server version up to a few months old or so. So we cannot expect this property to be present in `InteractiveGoal`s arriving from the server, and it must be optional.